src/wp-content/themes/twentynineteen/inc/template-functions.php

```diff
@@ -35,12 +35,12 @@ function twentynineteen_body_classes( $classes ) {
 /**
  * Adds custom class to the array of posts classes.
  */
-function twentynineteen_post_classes( $classes, $css_class, $post_id ) {
+function twentynineteen_post_classes( $classes ) {
 	$classes[] = 'entry';
 
 	return $classes;
 }
-add_filter( 'post_class', 'twentynineteen_post_classes', 10, 3 );
+add_filter( 'post_class', 'twentynineteen_post_classes' );
```

Comment:
It _might_ be worth showing the defaults `, 10, 1`—for consistency with `twentynineteen_post_thumbnail_sizes_attr`—because this changes the number of `$accepted_args`.

Reply:
Taking a look through the file:
- `body_class` just above this, and two other instances just call `add_filter( 'hook', 'callback' )`.
- This makes `twentynineteen_post_thumbnail_sizes_attr` the outlier by showing the defaults.
- For consistency throughout the file, it might be worth making `twentynineteen_post_thumbnail_sizes_attr` consistent with the rest of the `add_filter()` calls by removing `10, 1`?

Reply:
Consistency that way works, too :)

Reply:
The feedback addressed in https://github.com/WordPress/wordpress-develop/pull/4225/commits/dec4b321f99acf4c62dfe2d9f7e2513ebeaf26e3